test/Dapr.Client.Test/DistributedLockApiTest.cs

```diff
@@ -0,0 +1,84 @@
+// ------------------------------------------------------------------------
+// Copyright 2022 The Dapr Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ------------------------------------------------------------------------
+
+using System.Threading.Tasks;
+using Autogenerated = Dapr.Client.Autogen.Grpc.v1;
+using Xunit;
+using FluentAssertions;
+using System;
+
+namespace Dapr.Client.Test
+{
+    [System.Obsolete]
+    public class DistributedLockApiTest
```

Comment:
I think it's worth it to make sure the call fails if it's missing the correct parameters. Especially the `lockDuration` since we define that logic ourselves instead of using the `ArgumentVerifier`.